schema/deploy/tables/revision_status.sql

```diff
@@ -0,0 +1,21 @@
+-- Deploy cif:tables/revision_status to pg
+
+begin;
+
+alter table cif.amendment_status rename to revision_status;
+alter table cif.revision_status add column is_amendment_specific boolean;
+
+update cif.revision_status set is_amendment_specific = true where name='Draft' or name='Applied';
+update cif.revision_status set is_amendment_specific = false where name not in('Draft', 'Applied');
+
+alter table cif.project_revision rename amendment_status to revision_status;
+
+alter table cif.project_revision disable trigger _100_committed_changes_are_immutable, disable trigger _100_timestamps;
+update cif.project_revision set revision_status='Applied' where revision_status='Approved';
```

Comment:
Is `Approved` status being renamed to `Applied` or are they separate statuses?

Reply:
Renamed